Complete pagination Taxonomy-Ajax

nazmulhasan103 opened the pull request:
## PR Type
What kind of change does this PR introduce?
<!-- Mark completed items with an [x] -->
- [ ] Bugfix
- [ ] Security fix
- [ ] Improvement
- [x] New Feature
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Text changes
- [ ] Other... Please describe:

## Description
How to reproduce the issue or how to test the changes

1. check categories pagination
2. check locations pagination
3.

## Any linked issues
Fixes #

## Checklist

- [ ] My code follows the [WordPress coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)

obiPlabon committed:
Fix category checkbox field visibility issue with ajax search

Armanul46 committed:
Fixed listing expired status issue & improve listing expired notice in dashboard (#2194)

* sync

* fixed listing expired status issue

Armanul46 committed:
Fixed listing expired status issue & improve listing expired notice in dashboard (#2194)

* sync

* fixed listing expired status issue

nazmulhasan103 committed:
Complete pagination Taxonomy-Ajax

nazmulhasan103 committed:
Removed unnecessery class name

nazmulhasan103 committed:
Updated template version

nazmulhasan103 committed:
solved conflict

nazmulhasan103 committed:
solved conflict

nazmulhasan103 committed:
Merge branch 'trunk' into taxonomy-ajax

Armanul46 committed:
Fixed/single taxonomy issue (#2188)

* sync

* fixed single taxonomy issue

Armanul46 committed:
Replace Twitter text with X (#2191)

* sync

* Replace Twitter with X

MahfuzulAlam committed:
make order summery string translatable (#2192)

* make order summery string translatable

* fixed the spelling

RabbiIslamRony committed:
Update: Single listing title tag from `<h2>` to `<h1>` for better SEO also Modal Heading  (#2195)

* Update listing title markup for SEO

* replace Modal Heading to normal div

RabbiIslamRony committed:
Fix: Update search form number field dropdown placeholder based on us… (#2197)

Fix: Update search form number field dropdown placeholder based on user-provided details

obiPlabon committed:
Fix listing directory update on quick and bulk edit screen

RabbiIslamRony committed:
Update: Add Addon Kit to our Directorist child extension in the WordP… (#2202)

Update: Add Addon Kit to our Directorist child extension in the WordPress plugin list.

obiPlabon commented on includes/asset-loader/localized_data.php:
```diff
@@ -13,6 +13,11 @@ public static function load_localized_data() {
+			'ajax_url' => admin_url('admin-ajax.php'),
+			'nonce'    => wp_create_nonce('directorist_ajax_nonce'),
+		));
+
```
Please remove this and instead use the `directorist.ajax_url` and `directorist.directorist_nonce`

obiPlabon commented on assets/src/js/public/components/categoryLocation.js:
```diff
@@ -30,4 +32,62 @@ window.addEventListener('load', () => {
+            dataType: 'json',
+            data: {
+                action: 'directorist_taxonomy_pagination',
+                nonce: taxonomyPaginationAjax.nonce,
```
Use `directorist.directorist_nonce`

obiPlabon commented on assets/src/js/public/components/categoryLocation.js:
```diff
@@ -30,4 +32,62 @@ window.addEventListener('load', () => {
+        const attrs = $(current.closest(selector)).data('attrs');
+
+        $.ajax({
+            url: taxonomyPaginationAjax.ajax_url,
```
Use `directorist.ajax_url`

obiPlabon commented on assets/css/admin-main.css:
```diff
@@ -5817,8 +5817,8 @@ body.wp-admin {
 #atbdp_ie_download_sample:hover {
-  border-color: #264ef4;
-  background: #264ef4;
+  border-color: rgb(37.7561576355, 78.2783251232, 243.7438423645);
```
Why these color codes are so odd! `37.7561576355`!

iamibrahimriaz committed:
Custom Range Slider (console.error) - Fixed (#2204)

* Custom Range Slider Issue - Fixed

* max-value added on range-slider

* Number Field Range - Updated

* Range Slider AJAX request triggering multiple times when keyup value - Prevented

* Range Slider console.error - Fixed

Data was not properly found in php. That's why returns error message as value.

* range-slider.js called inside search-form.js - removed

---------

Co-authored-by: Rabbi Islam Rony ⚡️ <35329385+RabbiIslamRony@users.noreply.github.com>
Co-authored-by: Mehedi Hasan  ⚡️ <43073560+HeyMehedi@users.noreply.github.com>

obiPlabon committed:
Add users query limit to published listings owner only (#2198)

Co-authored-by: Mehedi Hasan  ⚡️ <43073560+HeyMehedi@users.noreply.github.com>

HeyMehedi committed:
fix: view count reset issue (#2207)

nazmulhasan103 committed:
used existing nonce

nazmulhasan103 committed:
improved code & fixed pagination content issue

nazmulhasan103 committed:
wrapped pagination under taxonomy items

nazmulhasan103 committed:
solved conflict

nazmulhasan103 committed:
solved conflict

nazmulhasan103 committed:
Merge branch 'trunk' into taxonomy-ajax

obiPlabon reviewed (approved):
Looks good
